README.md

```diff
@@ -73,9 +73,18 @@ Below you can find some common, general examples on how to use the plugins. Plea
 Excluding a directory:
 
 ```js
-// Exclude /images/
+// Exclude /hero/ directory
 [withRasterImages({
-    exclude: /images\/.*$/,
+    exclude: /hero\/.*$/,
+})],
```

Comment:
I would use something more descriptive and not so specific, like you had before. Just my opinion 😅